packit_service/constants.py

```diff
@@ -289,3 +289,8 @@ def from_number(number: int):
     "of the `pull_from_upstream` job. We believe this adjustment will simplify the onboarding "
     "process and enhance the overall user experience. "
 )
+
+
+class KojiAllowedAccountsAlias(Enum):
+    all_admins = "all_admins"
+    all_committers = "all_committers"
```

Comment:
I'm just wondering about the `all_` prefix. An alias is technically a project-specific virtual group, and if we use a `@` symbol for groups, we could use something similar for the aliases.

Reply:
personally I like the original proposal, having it like the current implementation, but no strong opinion. @packit/the-packit-team anyone has any preferences?

Reply:
I don't have strong opinions either.
But I wouldn't use the @ symbol. I agree that we have created a group, but it is a "packit" group and it will not show up in the fedoraproject UI; for this reason I think it is misleading to use @
We can have another symbol but it will not "immediately suggest" to the user the "group idea", so I don't know if it is worth having it?

Reply:
I just think that using a symbol is a little more consistent and gives the feeling that it's some special identifier, on the other hand `all_admins` is also pretty self-explanatory:

```yaml
allowed_pr_authors:
    - $admins
    - @packager
    - packit-stg
```

```yaml
allowed_pr_authors:
    - all_admins
    - @packager
    - packit-stg
```

I'm fine with both.

Reply:
If we used `$`, we could call it a variable rather than alias: `$admins expands to all users with admin access to the project`

Reply:
We already have the aliases for `targets` and these also do not use any special symbols. So if we would like to be consistent, I would go with the original idea.